lib/third_party/mbedtls/library/x509_crt.c

```diff
@@ -2014,7 +2014,7 @@ static int x509_crt_find_parent_in(
 {
     int ret;
     mbedtls_x509_crt *parent, *fallback_parent;
```

Comment:
There is also a path where parent is accessed unitialized, specifically when candidates is passed in as null.

Seems to me the best approach here is to initialize all of these locals here?